dist/index.html

```diff
@@ -6,20 +6,35 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;700&family=Playfair+Display:wght@600&display=swap" rel="stylesheet">
   <title>Overlook</title>
 </head>
 <body>
-  <!-- <img src="./images/booking-page.jpg" alt="" class="cover"> -->
-  <div class="cover"></div>
-  <section class="banner">
+  <section class="modal-login">
+    <div class="img-container">
+      <img src="./images/login-page.jpg" alt="privete pergolas inset around an infinity pool overlooking the ocean at dusk" class="cover-login">
+    </div>
+    <div class="inner-modal-login">
+      <h1 class="hotel-title">OVERLOOK<br><span class="logo-login">BOUTIQUE HOTEL</span></h1>
+      <form id="login" action="../src/scripts.js">
+        <input type="text" id="username" name="username" aria-label="enter username" placeholder="Username">
+        <input type="password" id="password" name="password" aria-label="enter password" placeholder="Password">
+        <button type="submit" form="login" value="submit" class="login-btn">Login</button>
+      </form>
+      <p class="false-validation"></p>
+    </div>
+  </section>
+  <div class="cover hidden"></div>
```

Comment:
It's difficult for me to tell what's going on with this `.cover` element, but I think it's housing the nice background image on the dashboard view - any reason this image isn't persisting between `nav-tab` clicks?  I think it would still look nice behind the past/upcoming trips views.

Reply:
This was a design decision. But yes, it gets hidden so that the user can focus on the rooms